apps/rush/src/start-dev.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+// This file is used during development to load the built-in plugins and to bypass
+// some other checks
+
+import * as rushLib from '@microsoft/rush-lib';
+import { PackageJsonLookup, Import } from '@rushstack/node-core-library';
+
+import { MinimalRushConfiguration } from './MinimalRushConfiguration';
+import { RushCommandSelector } from './RushCommandSelector';
+
+const builtInPluginConfigurations: rushLib._IBuiltInPluginConfiguration[] = [];
+
+function tryIncludePlugin(pluginName: string): void {
```

Comment:
Why is it called "try" include plugin?  What happens if the plugin cannot be resolved?

Given that the plugins are dev dependencies, is there even a valid scenario where the plugin should NOT get loaded?

Reply:
Ah meant to rename that.